Tests/test_PDB.py

```diff
@@ -1332,6 +1333,24 @@ def test_NACCESS_asa_file(self):
             naccess = process_asa_data(asa)
         self.assertEqual(len(naccess), 524)
 
+class TestPDBIO(unittest.TestCase):
+    
+    def test_example(self):
+        p = PDBParser(PERMISSIVE=True)
+        s = p.get_structure("test", sys.argv[1])
+        io = PDBIO()
+        io.set_structure(s)
+        filenumber, filename = tempfile.mkstemp()
+        os.close(filenumber)
+        with open("out2.pdb", "w") as fp:
+            s1 = p.get_structure("test1", sys.argv[1])
```

Comment:
You are still using ``sys.argv[1]`` and ``sys.argv[2]`` for file names. You need to use two automatically named temporary files here (a few lines above you setup a single temporary filename but never used it).